host/repo_config_info.h

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#ifndef OE_REPOSITORY_CONFIG_H
+#define OE_REPOSITORY_CONFIG_H
+
+// The following defines will be overwritten by the configuration
```

Comment:
Don't check in files that are never used. `gen_repo_config_header.sh` force deletes and generates this at build time anyway.

This comment is probably useful where the header is included instead (host/traceh.c)

Reply:
Removed